js/templates/listingCard.html

```diff
@@ -42,7 +42,10 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <div class="userIcon disc clrBr2 clrSh1"
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
```

Comment:
Since this is in a data attribute used for the content of a tooltip, we can't emojify it. As far as I can tell, it looks identical to the emojified version on Windows, we could probably leave this as-is, unless there's a big difference on other OSes. 

@morebrownies what do you think?